civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -7475,4 +7460,4 @@ function get_civihr_version() {
   }
 
   return $civihrVersion;
-}
+}
```

Comment:
missing newline at the end of the file (TIP: we have an .editorconfig file in this repo that, among other things, helps you to always have a newline at the end of the file. If you use PhpStorm, please install the editorconfig plugin to make this work)